template/build/core/phing/tasks/deploy.xml

```diff
@@ -99,11 +106,13 @@
 
     <!-- Copy required files from docroot. -->
     <copy todir="${deploy.dir}/docroot" overwrite="true">
-      <fileset dir="${docroot}">
+      <fileset dir="${docroot}" defaultexcludes="false">
```

Comment:
Rather than disabling all of these default excludes, I think we should simply include .gitignore files. E.g., `<include name="**/.gitignore"/>`. This seems like a more precise way to accomplish the end goal.